dockerversion/version_lib.go

```diff
@@ -14,4 +14,5 @@ const (
 	RuncCommitID       = "library-import"
 	InitCommitID       = "library-import"
 	PlatformName       = ""
+	ProductName        = ""
```

Comment:
Better to have a default value? (`moby`? `moby-engine`? `dockerd`?)

Reply:
I think it's better to keep it empty, because there are no "moby releases". Docker will set this.